marvin-cli/marvin_cli/management/engine.py

```diff
@@ -84,8 +89,8 @@ def import_project(file, dest):
     extract_folder(file, dest)
 
 @cli.command("engine-dryrun", help="Run engines in a standalone way.")
-@click.option('--host', '-h', prompt='gRPC host', help='gRPC Host Address', default='localhost')
-@click.option('--port', '-p', prompt='gRPC port', help='gRPC Port', default='50057')
+@click.option('--host', '-gh', help='gRPC Host Address', default='localhost')
```

Comment:
I thought `--host` will just be `-h` while `--grpchost` be `-gh`?
Should we change this `--host` to `--grpchost` as well?

Reply:
Yes, I think. Sorry for this, probably a copy and paste mistake. But we could change this since `engine-httpserver` uses `--host` as well, so `--grpchost` can be the standard for daemon host.